src/main/resources/templates/index.html

```diff
@@ -120,14 +167,17 @@
       plugins: [
         SwaggerUIBundle.plugins.DownloadUrl,
         pinLoginPlugin,
-        clearValidator
+        clearValidator,
+        insertClientIdsPlugin
       ],
       layout: 'StandaloneLayout',
       displayOperationId: true,
+      displayRequestDuration: true,
       docExpansion: 'none',
-      defaultModelsExpandDepth: -1, // hide the huge list of schemas under the routes
+      // defaultModelsExpandDepth: -1, // hide the huge list of schemas under the routes
```

Comment:
Did you mean to leave this commented out?

Reply:
whoops, I did not!